demo/priv/repo/migrations/20221221111118_create_users.exs

```diff
@@ -7,7 +7,7 @@ defmodule Demo.Repo.Migrations.CreateUsers do
       add(:first_name, :string, null: false)
       add(:last_name, :string, null: false)
       add(:role, :string, null: false)
-      add(:avatar, :string, null: false, default: "")
+      add(:avatar, :string, null: true, default: "")
```

Comment:
can we remove the default value and use `null` if no avatar is present?

Reply:
Yep, changed it.